src/components/common/Badge/RelationshipBadge.jsx

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+
+export function RelationshipBadge({ type, children }) {
+  const baseStyle =
+    'inline-flex items-center justify-center rounded-lg px-2 py-0.5 font-16-regular';
+
+  const variantStyles = {
+    friend: 'bg-blue-100 text-blue-600',
+    family: 'bg-green-100 text-green-600',
+    coworker: 'bg-purple-100 text-purple-600',
+    other: 'bg-orange-100 text-orange-600',
+  };
+
+  return (
+    <span
+      className={[baseStyle, variantStyles[type]].filter(Boolean).join(' ')}>
+      {children}
+    </span>
+  );
+}
+
```

Comment:
@heoc0523
1. 화살표 함수를 사용해주세요.
2. cva 패키지와 유틸함수 cn을 사용하시면 좋을 것 같습니다!
3. api 문서 확인해보시면 `"relationship": "친구"`, 요런 형태로 데이터를 받아와서 releationship을 badge 텍스트로 전달하고,
relationship의 값에 따라 style을 적용하게 설정하면 더 좋을 것 같습니다!

아래는 1번 2번을 사용한 방법 예시입니다.

```suggestion
const RelationshipBadge = ({ type, children }) => {
  const BadgeStyle = cva(
    'inline-flex items-center justify-center rounded-lg px-2 py-0.5 font-16-regular',
    {
      variantStyles: {
        friend: 'bg-blue-100 text-blue-600',
        family: 'bg-green-100 text-green-600',
        coworker: 'bg-purple-100 text-purple-600',
        other: 'bg-orange-100 text-orange-600',
      },
    }
  );

  return (
    <span className={cn(BadgeStyle({type}))}>
      {children}
    </span>
  );
};
```